sources/reference/admiral-aws-marketplace.md

```diff
@@ -80,3 +80,28 @@ $ ssh -L 50003:1.2.3.4:50003 -C ubuntu@1.2.3.4
 - Click `Install`. This should set up Shippable services and bring up the UI.
   Once complete, you should be able to navigate to `http://1.2.3.4:50001` and
   log into Shippable using a Github account.
+
+
+### Setup superuser account
+
+Shippable UI also provides an admin panel which helps users to
+
+- monitor builds across the system
+- manage build nodes for either the system or individual subscriptions
+- manage user accounts across the system
+- view and manage any build, project and subscription
+- Perform other housekeeping tasks
+
+Only the accounts that have been added to superuser list will be able to access
+this panel.
+
+- After successfully logging in using Github, navigate to the accounts page and copy the account ID
+
+- Navigate back to Admiral UI and scroll down to `Manage System SuperUsers`
+  sub-section in `Add-ons`
+  - paste the account ID from the previous step into the text box and hit `Add`
```

Comment:
I would say "from the account's page," or however you decide to describe it, here instead of "previous step" in case someone isn't reading very closely.